buildtest/settings/config.yml

```diff
@@ -31,7 +31,8 @@ system:
           shell: zsh
     # compiler block
     compilers:
-      modulepath: ['/tmp', '/var/']
+      # a list of directories to search for modules
+      # modulepath: ['/tmp', '/var/']
```

Comment:
can you add a new line between `modulepath` and the next statement so its clear the comments are not next to command.